lib/matplotlib/__init__.py

```diff
@@ -1347,12 +1347,15 @@ def tk_window_focus():
 
 
 def _init_tests():
-    try:
-        import faulthandler
-    except ImportError:
-        pass
-    else:
-        faulthandler.enable()
+    # work around https://bugs.python.org/issue31701
+    # faulthandler dumps 'Windows fatal exception: code 0xe06d7363'
+    if not (sys.platform == 'win32' and sys.version_info[:3] == (3, 6, 3)):
+        try:
+            import faulthandler
+        except ImportError:
+            pass
+        else:
+            faulthandler.enable()
```

Comment:
I would remove this completely and enable it on travis with `pytest-faulthandler`.

Reply:
even better (optional) if you document it at http://matplotlib.org/devdocs/devel/testing.html#requirements

Reply:
Not sure I follow. Faulthandler is useful, just not the current version or for cases like #9176

Reply:
1. It's output is not useful if you do not have debug symbols (distutils does not produce it for windows and strips for linux by default on non-debug version)
2. `faulthandler` could be enabled with pytest by `pytest-faulthandler` module (that's what I've suggested earlier)

Reply:
I do have debug symbols and use them frequently when testing matplotlib. `pytest-faulthandler` is yet another dependency. I'll leave it to the maintainers. This PR is to check if the build and tests succeed on Appveyor with Python 3.6.3.

Reply:
The build would pass even without disabling `faulthandler`.

Reply:
Yes, disabling `faulthandler` is not related to building. Still, Python 3.6.3 now uses the vc141 toolchain if installed and I would like to see if that affects matplotlib.